scripts/in_container/check_environment.sh

```diff
@@ -21,6 +21,21 @@ EXIT_CODE=0
 
 DISABLED_INTEGRATIONS=""
 
+# We want to perform only forward lookup of the service IP address. Netcat when run without -n
+# Performs both forward and reverse lookup and fails if the reverse lookup name does not match
+# original name even if the host is reachable via IP. This happens randomly with docker-compose
+# in Github Actions (apparently it is some race condition when one service starts before DNS)
+# Since we are not using reverse lookup elsewhere we can perform forward lookup in python
+# and use the IP in NC and add '-n' switch to disable any DNS use.
```

Comment:
```suggestion
# We want to perform only forward lookup of the service IP address to remove the irrelevant
# log message - "DNS fwd/rev mismatch"
#
# Long explanation:
# Netcat when run without -n, performs both forward and reverse lookup and fails if the reverse lookup name
# does not match original name even if the host is reachable via IP. Then it displays message 
# "DNS fwd/rev mismatch",  which introduces additional noise in the log.
# To mitigate, we are not using reverse lookup elsewhere we can perform forward lookup in python
# and use the IP in NC and add '-n' switch to disable any DNS use.
```

Reply:
I updated the description with a little different emphasis.